examples/flash_with_rtic.rs

```diff
@@ -45,258 +33,74 @@ mod app {
         let cp = cx.core;
 
         let rcc = dp.RCC.constrain();
-        let mut pll_config = stm32g4xx_hal::rcc::PllConfig::default();
-
-        // Sysclock is based on PLL_R
-        pll_config.mux = stm32g4xx_hal::rcc::PLLSrc::HSI; // 16MHz
-        pll_config.n = stm32g4xx_hal::rcc::PllNMul::MUL_32;
-        pll_config.m = stm32g4xx_hal::rcc::PllMDiv::DIV_2; // f(vco) = 16MHz*32/2 = 256MHz
-        pll_config.r = Some(stm32g4xx_hal::rcc::PllRDiv::DIV_2); // f(sysclock) = 256MHz/2 = 128MHz
```

Comment:
Correct me if I am wrong but the clock settup is not really needed for this example, right? Thought I'd remove all but the essentials to give more focus on the flash related things